test/features/step_definitions/poa_steps.rb

```diff
@@ -0,0 +1,90 @@
+require_relative '../helpers/i18n_helper.rb'
+
+i18n = I18nHelper.new
+
+Given(/^I navigate to the SDK with PoA feature enabled$/) do
+  open_sdk(@driver, { 'poa' => true, 'useWebcam' => false })
+end
+
+Then(/^I navigate to document selection screen and verify UI elements$/) do
+
+  bank_statement_cell = 'Bank/Building Society Statemente-statements accepted'
```

Comment:
Let's create a ticket to extract data from the step logic. Apart from that it looks good!

Reply:
Since we move towards tests written in JS, I think we should rather focus on rewriting these. Our new JS framework won't involve any steps or feature files :)